video_xblock/video_xblock.py

```diff
@@ -161,6 +171,20 @@ def validate_field_data(self, validation, data):
         """
         Validate data submitted via xblock edit pop-up
         """
+        if data.account_id and data.player_id:
+            try:
+                r = requests.head(VideoXBlock.get_brightcove_js_url(data))
+                if r.status_code != 200:
+                    validation.add(ValidationMessage(
+                        ValidationMessage.ERROR,
+                        _(u"Invalid Player Id, please recheck")
+                    ))
+            except requests.ConnectionError:
+                validation.add(ValidationMessage(
+                    ValidationMessage.ERROR,
+                    _(u"Connection to https://www.brightcove.com error.")
```

Comment:
"Can't validate submitted player id. Please try to save settings one more time."